### PR TITLE
Add Language Options to Settings

### LINKED ISSUE #4068: Add Spanish, French, & Italian to Language options in the Settings panel
image

### PR DESCRIPTION
**Pull request by joeizang**
fixes #4068

### CODE REVIEW

File: apps/remix-ide/src/app/tabs/locale-module.js [hunk 1]
```diff
@@ -5,11 +5,15 @@ import * as packageJson from '../../../../../package.json'
...
 import enJson from './locales/en'
 import zhJson from './locales/zh'
+import esJson from './locales/es'
+import frJson from './locales/fr'
 const _paq = window._paq = window._paq || []
 
 const locales = [
   { code: 'en', name: 'English', localeName: 'English', messages: enJson },
   { code: 'zh', name: 'Chinese Simplified', localeName: '简体中文', messages: zhJson },
+  { code: 'es', name: 'Spanish', localeName: 'Español', messages: esJson },
```
Comment (ryestew) [inline apps/remix-ide/src/app/tabs/locale-module.js:15]: These should be alphabetized - well at least alphabetized in English - so the first entry of this list would be chinese.

**Review by ryestew — CHANGES_REQUESTED** [review]
I would like this PR to include the upcoming .json files - in Spanish and French and maybe in Chinese.  These additions could be in a different PR.  I'll coordinate with you about the timing of when these translations will be ready.

File: apps/remix-ide/src/app/tabs/locale-module.js (hunk 1 above)
Comment (joeizang) [reply apps/remix-ide/src/app/tabs/locale-module.js:15]: done

**Review by ryestew — APPROVED** [review]

**Comment by ryestew** [comment]
we can add Italian to this list

**Pull request merged**